### ISSUE TITLE
"Library" feature not documented

### ISSUE THREAD

**Issue by pepe82sh** [toniebox-reverse-engineering/teddycloud#123]
I'm getting used to teddycloud, having 2 tonie boxes connected right now. Now I want to start serving content for creative tonies from teddycloud rather than the official tonies server. I assume the Library is where I should go to, but I'm not sure how to use it. I can't find it in the wiki, nor on the GitHub page. Any hint would be very welcome. Sorry if there is something I overlooked.

**Comment by SciLor** [comment]
Just some general:
### Content
Content replicates the structure of the SD:/CONTENT/ folder of the box. TeddyCloud serves its content from this directory structure. There no nice names are possible

### Library
The library is just a bunch of random files. Using the "Assign Unknown" feature will serve the marked content to the first tag asking for content (with no content in the content directory). 
This feature also works in the content folder, but there you won't have nice names.

### Future ideas
For the future, we might want to add features that allow linking the library file with the actual content file, without coping and therefore duplicating the content. 
We also want to cache all files to the library folder that are going through the cloud for easier reuse.

**Comment by oliver-koch** [comment]
I had a similar question regarding Settings and the Library. 

https://forum.revvox.de/t/teddycloud-webinterface-settings/152

**Comment by oliver-koch** [comment]
Tldr:

What is the libray used for?

The content folder is the folder used to provide data for the box. 
The library is just a storage for random files, so you may put your TAF collection with nice names there.

**Comment by pepe82sh** [comment]
Awesome, thanks for the quick response! I'll check out the forum, I haven't found that resource before :)
